### DISCUSSION TITLE
Sequential models in DeepXDE

### DISCUSSION THREAD

**Discussion by dang-trinh156** [lululxvi/deepxde#1647] [Q&A]
Dear Dr.Lulu and developer team and users,
I've scanned through the FAQ and search on this discussion session. All what I can find is this post:https://github.com/lululxvi/deepxde/discussions/1488.
Also, I've read the source code in Geometry, icbc, PDE, and they're all implemented on 2D-arrays (i.e. [batch_size or n_data,  spatial-temporal features]). Here's an example:
![image](https://github.com/lululxvi/deepxde/assets/124559501/b9be4d78-afe1-483b-926f-16eb8f9ade66)
The error function returns an array/tensor in 2D. Therefore, only FCNN architectures are used.

Question:
Have there been any examples, or snippets of code about the implementation of 3D inputs (i.e. [batch_size, sequence length, features]) for sequential models?

Thank you in advance!

**Comment by lululxvi** [comment]
No. Then you need to implement the network by yourself.
